core/templates/dev/head/pages/base.html

```diff
@@ -165,6 +165,38 @@ <h2>
 
     {% include 'pages/footer_js_libs.html' %}
 
+    <!-- About-page -->
+    <script src="/templates/dev/head/pages/about/about-page.module.js"></script>
+    <script src="/templates/dev/head/pages/about/about-page.controller.js"></script>
+
+    <!-- donate page -->
+    <script src="/templates/dev/head/pages/donate/donate-page.module.js"></script>
+    <script src="/templates/dev/head/pages/donate/donate-page.controller.js"></script>
+
+    <!-- creator dashboard page -->
+    <script src="/templates/dev/head/pages/creator_dashboard/creator-dashboard.module.js"></script>
+    <script src="/templates/dev/head/pages/creator_dashboard/creator-dashboard.controller.js"></script>
+
+    <!-- error page -->
+    <script src="/templates/dev/head/pages/error/error.module.js"></script>
+    <script src="/templates/dev/head/pages/error/error.controller.js"></script>
+
+    <!-- collection player page -->
+    <script src="/templates/dev/head/pages/collection_player/collection-player.module.js"></script>
```

Comment:
Are these scripts required here as well as in collection-player.mainpage.html?

Reply:
I am not sure, (comment link https://github.com/oppia/oppia/pull/6524#discussion_r278119285). I will try to remove them and push the changes if those pass tests then ill keep them removed.

Reply:
done

Reply:
Sure, check it once and do let me know the issues you face.

Reply:
They are failing all tests, so I think that was correct, the files do not get loaded twice. Can I revert the commit now?